Improve the last sentence in the 'Try it in staging first' section

cavcrosby opened the pull request.

cavcrosby commented:
I am now seeing the first word capitalization was fixed as part of https://github.com/ansible/ansible-documentation/commit/d9377260c04c1b17414cc2baa3c48e91f2571cc0. The colon addition may still be warranted.

oraNod commented:
> I am now seeing the first word capitalization was fixed as part of [d937726](https://github.com/ansible/ansible-documentation/commit/d9377260c04c1b17414cc2baa3c48e91f2571cc0). The colon addition may still be warranted.

@cavcrosby Hey, thanks for the doc fix! Yes it looks like the capitalization was fixed in a different commit. If you could resolve that conflict and update this PR I think it's worth adding the colon and maybe even a phrase that leads in to the code block. Something like `flag ``--syntax-check`` as in the following example:` might work.

cavcrosby committed:
Improve the 'Try it in staging first' section

The last sentence in the the 'Try it in staging first' section has been
appended with additional verbiage to follow other sentences that contain
a subsequent example code snippet.

oraNod reviewed (approved):
LGTM. Thanks for the docs contribution @cavcrosby

samccann commented on docs/docsite/rst/tips_tricks/ansible_tips_tricks.rst:
```diff
@@ -133,7 +133,7 @@ Try it in staging first
 
 Testing changes in a staging environment before rolling them out in production is always a great idea.
-Your environments need not be the same size, and you can use group variables to control the differences between environments. You can also check for any syntax errors in the staging environment using the flag ``--syntax-check``
+Your environments need not be the same size, and you can use group variables to control the differences between environments. You can also check for any syntax errors in the staging environment using the flag ``--syntax-check`` like in the following example:
```
```suggestion
Your environments need not be the same size, and you can use group variables to control the differences between environments. You can also check for any syntax errors in the staging environment using the flag ``--syntax-check`` such as in the following example:
```

samccann committed:
Update docs/docsite/rst/tips_tricks/ansible_tips_tricks.rst
